### PR TITLE
gps_compass: add Holybro Unicore GPS module

### PR DESCRIPTION
**Pull request by julianoes**
This adds a page about the Holybro GPS based on the UM982 module.

The current state is still work in progress until we have the definite title and shop links.

### CODE REVIEW

File: en/gps_compass/README.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ Device | GPS | Compass | [RTK](../gps_compass/rtk_gps.md) | [GPS Yaw Output](#co
 [Holybro H-RTK F9P Helical or Base](../gps_compass/rtk_gps_holybro_h-rtk-f9p.md) | F9P | IST8310 | &check; | | &check;|
 [Holybro H-RTK F9P Rover Lite](../gps_compass/rtk_gps_holybro_h-rtk-f9p.md) | F9P | IST8310 | &check; | | |
 [Holybro H-RTK M8P GNSS](../gps_compass/rtk_gps_holybro_h-rtk-m8p.md) | M8P | IST8310 | &check; | | |
+[Holybro Unicore GPS](../gps_compass/gps_holybro_unicore.md) | UM982 | ? | &check; | | &check; |
```
Comment (hamishwillee) [inline en/gps_compass/README.md:52]: Reminder that we need the ? mark filled

File: en/gps_compass/gps_holybro_unicore.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Holybro Unicore UM982 GPS
+
+The [Holybro Unicore UM982 GPS](http://www.holybro.com/product/TODO/) is an multi-band high-precision [RTK GNSS System](../gps_compass/rtk_gps.md) series launched by Holybro.
```
Comment (hamishwillee) [inline en/gps_compass/gps_holybro_unicore.md:3]: Reminder need TODO on link filled

File: en/gps_compass/gps_holybro_unicore.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+## Where to Buy
+
+* [H-RTK F9P (Holybro Website)](https://shop.holybro.com/TODO)
+
+## Configuration
+
+The Unicore module talks the NMEA protocol extended with some proprietary Unicore sentences. The serial baudrate is 230400.
+
+In order to use it in PX4, the following parameters need to be set (parameters if connected to GPS 1):
```
Comment (hamishwillee) [inline en/gps_compass/gps_holybro_unicore.md:17]:
```suggestion
The following PX4 parameters [must be set](../advanced_config/parameters.md) (if connected to GPS 1):
```

File: en/gps_compass/gps_holybro_unicore.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+- [SER_GPS1_BAUD](../advanced_config/parameter_reference.md#SER_GPS1_BAUD) -> 230400
+- [GPS_1_PROTOCOL](../advanced_config/parameter_reference.md#GPS_1_PROTOCOL) -> 6: NMEA
+- [GPS_1_PROTOCOL](../advanced_config/parameter_reference.md#GPS_1_PROTOCOL) -> 6: NMEA
```
Comment (hamishwillee) [inline en/gps_compass/gps_holybro_unicore.md:21]: These two lines are the same

File: en/gps_compass/gps_holybro_unicore.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+## Enable GPS heading/yaw
```
Comment (hamishwillee) [inline en/gps_compass/gps_holybro_unicore.md:23]:
```suggestion
## Enable GPS Heading/Yaw
```

File: en/gps_compass/gps_holybro_unicore.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+The Unicore module comes with two antennas, a primary (right connector) and a secondary (left connector) antenna.
+
+- [EKF2_AID_MASK](../advanced_config/parameter_reference.md#EKF2_AID_MASK) -> Set bit 7 (128) to enable GPS yaw fusion, so to use the GPS heading
+- [GPS_YAW_OFFSET](../advanced_config/parameter_reference.md#GPS_YAW_OFFSET) -> Set heading offset to 0 if the primary antenna is in the front. The angle increases clock-wise, so set the offset to 90 degrees if the primary antenna is on the right side of the vehicle (and the secondary on the left side).
+
+
+## RTK corrections
```
Comment (hamishwillee) [inline en/gps_compass/gps_holybro_unicore.md:31]:
```suggestion
## RTK Corrections
```

File: en/gps_compass/gps_holybro_unicore.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+:::note
+The RTK functionality has not been tested and verified yet.
```
Comment (hamishwillee) [inline en/gps_compass/gps_holybro_unicore.md:34]: When this occurs, there are more links that will need to be added for this - e.g. https://docs.px4.io/main/en/gps_compass/rtk_gps.html#supported-rtk-devices

File: en/gps_compass/gps_holybro_unicore.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+:::
+
+
+## Wiring
+
+The module comes with both GH 10-pin & 6-pin cables that are compatible with the GPS1 & GPS2 ports on flight controllers that use the Pixhawk Connector Standard, such as [Pixhawk 6x](../flight_controller/pixhawk6x.md) and [Pixhawk 6c](../flight_controller/pixhawk6c.md).
```
Comment (hamishwillee) [inline en/gps_compass/gps_holybro_unicore.md:40]:
```suggestion
The module comes with both GH 10-pin & 6-pin cables that are compatible with the GPS1 & GPS2 ports on flight controllers that use the [Pixhawk Connector Standard](https://github.com/pixhawk/Pixhawk-Standards/blob/master/DS-009%20Pixhawk%20Connector%20Standard.pdf), such as [Pixhawk 6x](../flight_controller/pixhawk6x.md) and [Pixhawk 6c](../flight_controller/pixhawk6c.md).
```

**Review by hamishwillee — COMMENTED** [review]
Looks great module a couple of subedits (fixed) and comments, most of which you already know about. Let's leave in draft until good to go.

File: en/gps_compass/gps_holybro_unicore.md (hunk 4 above)
Comment (julianoes) [reply en/gps_compass/gps_holybro_unicore.md:21]: Thanks, happy copy pasting.

Comment (julianoes) [reply en/gps_compass/gps_holybro_unicore.md:21]: Fixed.

File: en/gps_compass/gps_holybro_unicore.md (hunk 7 above)
Comment (julianoes) [reply en/gps_compass/gps_holybro_unicore.md]: Done.

**Comment by julianoes** [comment]
This should be done apart from shop links.

**Comment by hamishwillee** [comment]
Looks good. I did a minor structural update to put the wiring first. Generally I do it this way - I figure you connect it first. Then if you didn't connect to GPS1 (say) you know it when you get to the config bit. I also nested the PX4 config sections. Merging.

**Comment by vincentpoont2** [comment]
@hamishwillee This product has not be released. Should/can we put a label saying it has not been released? I will be adding additional information to this page next week.

**Comment by hamishwillee** [comment]
Yes. Done in https://github.com/PX4/PX4-user_guide/pull/2439

**Pull request merged**